migrations/versions/86c0f6b6a176_.py

```diff
@@ -0,0 +1,52 @@
+""" add type field to organisation model
+
+Revision ID: 86c0f6b6a176
+Revises: 7937dae319b5
+Create Date: 2021-02-09 03:29:03.763016
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from backend.models.postgis.statuses import OrganisationType
+
+
+# revision identifiers, used by Alembic.
+revision = "86c0f6b6a176"
+down_revision = "7937dae319b5"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("organisations", sa.Column("type", sa.Integer(), nullable=True))
+    op.execute("UPDATE organisations SET type = {}".format(OrganisationType.FREE.value))
+    op.alter_column("organisations", "type", nullable=False)
+    op.drop_index(
```

Comment:
Why do we have those drop / create index commands?

Reply:
They were auto-generated when the migration file was created. I figured the index was being dropped and recreated because of the change in the model/table. Should they be removed?

Reply:
I don't know. I don't see that behaviour on Django Migrations.